### PR TITLE
feat(hub): clickable order id

### PR DESCRIPTION
**Pull request by frenautvh**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/manager-hub
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | internal MANAGER-4645
| License          | BSD 3-Clause

## Description

Make last order clickable

### CODE REVIEW

File: packages/manager/modules/core/src/redirection/redirection.constants.js [hunk 1]
```diff
@@ -79,6 +81,7 @@ export default {
       },
     },
     help: 'https://help.ovhcloud.com',
+    order: `${managerRoot.CA}/manager/${URI.order}`,
```
Comment (antleblanc) [inline packages/manager/modules/core/src/redirection/redirection.constants.js]: Correct me if I'm wrong, isn't supposed to be:

```suggestion
    order: `${managerRoot.CA}/manager/dedicated/${URI.order}`,
```

Comment (frenautvh) [reply packages/manager/modules/core/src/redirection/redirection.constants.js]: @antleblanc i don't know, currently dedicated CA in on /manager, is it planned to be moved ?

Comment (antleblanc) [reply packages/manager/modules/core/src/redirection/redirection.constants.js]: From my point of view I would answer **Yes**.

cc @marie-j to confirm.

Comment (marie-j) [reply packages/manager/modules/core/src/redirection/redirection.constants.js]: @frenautvh This is future URL for hub, so dedicated in CA has to move

**Pull request merged**